### ISSUE TITLE
use --deployment option for bundler

### ISSUE THREAD

**Issue by mreinsch** [aws/opsworks-cookbooks#181]
we are running into some issues with bundler on production because the --deployment flag is missing when bundling. That's there: https://github.com/aws/opsworks-cookbooks/blob/release-chef-11.4/rails/libraries/rails_configuration.rb#L62 - should run "bundle install --deployment" with the rest of the params unchanged. See also http://bundler.io/v1.6/man/bundle-install.1.html#DEPLOYMENT-MODE

**Comment by mreinsch** [comment]
and also here: https://github.com/aws/opsworks-cookbooks/blob/release-chef-11.10/rails/libraries/rails_configuration.rb#L41

**Comment by awsrequena** [comment]
related to pull request https://github.com/aws/opsworks-cookbooks/pull/158

**Comment by kalbasit** [comment]
Yes same bug as #158, happens whenever you have a private gem from Github (or other sources) installed with Git.
